ROM0 not found

Trying to use the tool but whatever .mra file I use I always get this error:

```bash
➜  ghouls git:(master) ✗ ../mra Ghouls\'n\ Ghosts\ -World-.mra
src/rom.c:299: error: ROM0 not found in MRA.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,0 +1,11 @@
+FROM debian
+
+RUN apt update && apt -y install git make gcc zlib1g-dev
+
+RUN git clone https://github.com/sebdel/mra-tools-c.git
+RUN cd mra-tools-c && make
+COPY entry.sh mra-tools-c/entry.sh
+
+WORKDIR /mras
+
+ENTRYPOINT ["../mra-tools-c/entry.sh"]

--- a/README.md
+++ b/README.md
@@ -19,6 +19,22 @@ sudo apt install mingw-w64 libz-mingw-w64-dev
 make -f makefile.windows
 ```
 
+### Build using Docker
+
+In some platforms such as MacOS the tool needs to be used inside a linux container. Just clone the repo:
+```bash
+git clone https:\\https://github.com/sebdel/mra-tools-c.git
+```
+
+Copy all MRA files and zipped files in two directories and then:
+
+```bash
+cd mra-tools-c
+ROMS=/my/roms/directory  MRAS=/my/mra_files/directory docker-compose up --build mra
+```
+
+The .rom and .arc files will be generated inside your mra_files directory.
+
 ## License
 
 You do whatever you want with it, it does whatever it wants to your PC. Don't credit me, don't blame me :)

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,0 +1,11 @@
+version: "3"
+
+services:
+  mra:
+    build: .
+    image: mra
+    volumes:
+      - $ROMS:/roms
+      - $MRAS:/mras
+    environment:
+      - ROMS=$ROMS

--- a/entry.sh
+++ b/entry.sh
@@ -1,0 +1,8 @@
+#!/bin/bash
+for f in *.mra
+do
+    echo "Processing $f file..."
+    ../mra-tools-c/mra -A -z ../roms "$f"
+done
+echo "Generated ROMS:"
+ls -la *.rom

